deploy/WrappedERC20.js

```diff
@@ -1,15 +1,15 @@
+const { getWalletContract } = require("../utils/crossChainHelper")
 module.exports = async function ({ deployments, getNamedAccounts }) {
 	const { deploy } = deployments
 	const { deployer } = await getNamedAccounts()
 	console.log(`Deployer address: ${deployer}`)
 
-	const wrappedTokenBridge = await ethers.getContract("WrappedTokenBridge")
-	const wrappedTokens = ["WETH", "USDC", "USDT"]
+	const wrappedTokens = ["USDC"]
```

Comment:
we also want to add WETH?

Reply:
This is redundant script, we won't be needing this

Reply:
Ok so if we ate not using this please revert the changes